api/config/config.go

```diff
@@ -61,6 +63,17 @@ type (
 		Stack           string `yaml:"stack"`
 		StagingMemoryMB int    `yaml:"stagingMemoryMB"`
 	}
+
+	InfoConfig struct {
+		Build                    string                 `yaml:"build"`
+		Description              string                 `yaml:"description"`
+		Name                     string                 `yaml:"name"`
+		Version                  int                    `yaml:"version"`
+		MinCLIVersion            string                 `yaml:"minCLIVersion"`
+		MinRecommendedCLIVersion string                 `yaml:"minRecommendedCLIVersion"`
```

Comment:
According to the [CF API docs](https://v3-apidocs.cloudfoundry.org/version/3.164.0/index.html#get-platform-info), 

```
"cli_version": {
    "minimum": "6.22.0",
    "recommended": "latest"
  },
```

With respect to the `recommended` attribute above, it probably makes sense to rename `MinRecommendedCLIVersion` to `RecommendedCLIVersion`. Having the `Min` prefix is a bit misleading.

Reply:
In this case I've tried to mimic the previous behaviour of cloud_controller_ng. They are using min_recommended_cli_version for whatever reason:

https://github.com/cloudfoundry/cloud_controller_ng/blob/b970936bc611b97b9e248489baa4ff9bd4ed06e0/app/controllers/v3/info_controller.rb#L12
https://github.com/cloudfoundry/capi-release/blob/0abe5f7855c6004f17b0b61f944b3ae412cac78c/jobs/cloud_controller_ng/templates/cloud_controller_ng.yml.erb#L131

But it might be time to get rid of that confusion :-)

Reply:
I've removed the `min` prefix for the recommended CLI version